<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerDTISliceDisplayWidget</class>
 <widget class="qSlicerWidget" name="qSlicerDTISliceDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diffusion Tensor Volume Slice Display</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="GlyphVisibilityLabel">
     <property name="text">
      <string>Visibility:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="GlyphVisibilityCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="GlypthOpacityLabel">
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ctkSliderWidget" name="GlyphOpacitySliderWidget">
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="pageStep">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="GlyphColorTableLabel">
     <property name="text">
      <string>Scalar ColorMap:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="qMRMLColorTableComboBox" name="GlyphScalarColorTableComboBox">
     <property name="hideChildNodeTypes">
      <stringlist notr="true">
       <string>vtkMRMLDiffusionTensorDisplayPropertiesNode</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="GlyphColorByScalarLabel">
     <property name="text">
      <string>Color by Scalar:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="qMRMLScalarInvariantComboBox" name="GlyphColorByScalarComboBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="GlyphScalarRangeLabel">
     <property name="text">
      <string>Scalar Range:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="GlyphGeometryLabel">
     <property name="text">
      <string>Glyph Type:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="GlyphGeometryComboBox">
     <item>
      <property name="text">
       <string>Lines</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tubes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ellipsoids</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="GlyphScaleLabel">
     <property name="text">
      <string>Scale Factor:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="ctkSliderWidget" name="GlyphScaleSliderWidget">
     <property name="pageStep">
      <double>10.000000000000000</double>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="GlyphSpacingLabel">
     <property name="text">
      <string>Spacing:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="ctkSliderWidget" name="GlyphSpacingSliderWidget">
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QStackedWidget" name="GlyphAdvancedPropertiesWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="LinePropertiesPage">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="LineEigenVectorLabel">
         <property name="text">
          <string>Glyph EigenVector:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="LineEigenVectorComboBox">
         <item>
          <property name="text">
           <string>Major</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Middle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minor</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TubePropertiesPage">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="TubeEigenVectorLabel">
         <property name="text">
          <string>Glyph EigenVector:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="TubeEigenVectorComboBox">
         <item>
          <property name="text">
           <string>Major</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Middle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minor</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="EllipsoidPropertiesPage"/>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="GlyphManualScalarRangeCheckBox">
       <property name="toolTip">
        <string>Manual or Auto scalar range</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkRangeWidget" name="GlyphScalarRangeWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLScalarInvariantComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLScalarInvariantComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerDTISliceDisplayWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>GlyphScalarColorTableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GlyphManualScalarRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>GlyphScalarRangeWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GlyphGeometryComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>GlyphAdvancedPropertiesWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerDTISliceDisplayWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>GlyphScalarColorTableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
